tasks/extract-binaries-from-image/tests/mocks.sh

```diff
@@ -0,0 +1,16 @@
+# mocks to be injected into task step scripts
+
+set -x
```

Comment:
Note that this file will be injected to the top of the task script. So this means, that the shebang that is defined there will no longer apply. I'd say we want `-e` here, right? Technically, it will work that way, because after adding the mocks, the task will no longer have a shebang at the top, so tekton will use the default: https://tekton.dev/docs/pipelines/tasks/#running-scripts-within-steps

So while this will work, I'd suggest copying the same shebang as we have in the actual task:
```
#!/bin/sh -ex
```

(I am not 100 % how tekton determines if a script has a shebang or not - if empty lines/ comments at the top are ignored - I would expect that.)

Reply:
Done!

Reply:
I see `set -ex`. The suggestion was to copy the shebang, to have exactly the same in both real usage and mocked scenario.

Reply:
Done! I misunderstood, sorry!

Reply:
No worries, thanks.